.github/workflows/cypress.yml

```
@@ -0,0 +1,30 @@
+name: Cypress Tests
+
+on: [push, pull_request]
+
+jobs:
+  cypress-run:
+    name: Test
+    runs-on: ubuntu-18.04
```

```suggestion
    # matching current Netlify build environment
    runs-on: ubuntu-16.04
```
Netlify builds in 16.04, so I think it makes sense to match that.